bridge/src/main/java/com/livefront/bridge/ActivityLifecycleCallbacksAdapter.java

```diff
@@ -2,45 +2,43 @@
 
 import android.app.Activity;
 import android.app.Application.ActivityLifecycleCallbacks;
-import android.os.Build;
 import android.os.Bundle;
-import android.support.annotation.RequiresApi;
+import androidx.annotation.NonNull;
 
 abstract class ActivityLifecycleCallbacksAdapter implements ActivityLifecycleCallbacks {
 
-    @Override
-    public void onActivityCreated(Activity activity, Bundle savedInstanceState) {
-        // no-op
-    }
-
-    @Override
-    public void onActivityDestroyed(Activity activity) {
-        // no-op
-    }
-
-    @Override
-    public void onActivityPaused(Activity activity) {
-        // no-op
-    }
-
-    @Override
-    public void onActivityResumed(Activity activity) {
-        // no-op
-    }
-
-    @Override
-    public void onActivitySaveInstanceState(Activity activity, Bundle outState) {
-        // no-op
-    }
-
-    @Override
-    public void onActivityStarted(Activity activity) {
-        // no-op
-    }
-
-    @Override
-    public void onActivityStopped(Activity activity) {
-        // no-op
-    }
-
+  @Override
+  public void onActivityCreated(@NonNull Activity activity, Bundle savedInstanceState) {
+    // no-op
+  }
+
+  @Override
+  public void onActivityDestroyed(@NonNull Activity activity) {
+    // no-op
+  }
+
+  @Override
+  public void onActivityPaused(@NonNull Activity activity) {
+    // no-op
+  }
+
+  @Override
+  public void onActivityResumed(@NonNull Activity activity) {
+    // no-op
+  }
+
+  @Override
+  public void onActivitySaveInstanceState(@NonNull Activity activity, @NonNull Bundle outState) {
+    // no-op
+  }
+
+  @Override
+  public void onActivityStarted(@NonNull Activity activity) {
+    // no-op
+  }
+
+  @Override
+  public void onActivityStopped(@NonNull Activity activity) {
+    // no-op
+  }
```

Comment:
This is using the wrong formatting. We should be consistent with the rest of the project.

Reply:
My bad, my IDE using google-java-format by default. Did't know that there any roles about formating.

Reply:
Try importing this as a style.  I think its old but it might still work:

```
<code_scheme name="Bridge" version="173">
  <option name="RIGHT_MARGIN" value="100" />
  <option name="FORMATTER_TAGS_ENABLED" value="true" />
  <option name="KEEP_CONTROL_STATEMENT_IN_ONE_LINE" value="false" />
  <option name="KEEP_BLANK_LINES_IN_CODE" value="1" />
  <option name="BLANK_LINES_AROUND_FIELD" value="1" />
  <option name="BLANK_LINES_AFTER_CLASS_HEADER" value="1" />
  <option name="ALIGN_MULTILINE_PARAMETERS" value="false" />
  <option name="ALIGN_MULTILINE_FOR" value="false" />
  <option name="CALL_PARAMETERS_WRAP" value="1" />
  <option name="METHOD_PARAMETERS_WRAP" value="1" />
  <option name="EXTENDS_LIST_WRAP" value="1" />
  <option name="THROWS_LIST_WRAP" value="1" />
  <option name="EXTENDS_KEYWORD_WRAP" value="1" />
  <option name="THROWS_KEYWORD_WRAP" value="1" />
  <option name="METHOD_CALL_CHAIN_WRAP" value="1" />
  <option name="BINARY_OPERATION_WRAP" value="1" />
  <option name="BINARY_OPERATION_SIGN_ON_NEXT_LINE" value="true" />
  <option name="TERNARY_OPERATION_WRAP" value="1" />
  <option name="TERNARY_OPERATION_SIGNS_ON_NEXT_LINE" value="true" />
  <option name="FOR_STATEMENT_WRAP" value="1" />
  <option name="ARRAY_INITIALIZER_WRAP" value="1" />
  <option name="ASSIGNMENT_WRAP" value="1" />
  <option name="PLACE_ASSIGNMENT_SIGN_ON_NEXT_LINE" value="true" />
  <option name="WRAP_COMMENTS" value="true" />
  <option name="IF_BRACE_FORCE" value="3" />
  <option name="DOWHILE_BRACE_FORCE" value="3" />
  <option name="WHILE_BRACE_FORCE" value="3" />
  <option name="FOR_BRACE_FORCE" value="3" />
  <AndroidXmlCodeStyleSettings>
    <option name="USE_CUSTOM_SETTINGS" value="true" />
    <option name="LAYOUT_SETTINGS">
      <value>
        <option name="INSERT_LINE_BREAK_BEFORE_NAMESPACE_DECLARATION" value="true" />
      </value>
    </option>
  </AndroidXmlCodeStyleSettings>
  <JavaCodeStyleSettings>
    <option name="FIELD_NAME_PREFIX" value="m" />
    <option name="STATIC_FIELD_NAME_PREFIX" value="s" />
    <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="9999" />
    <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="9999" />
    <option name="PACKAGES_TO_USE_IMPORT_ON_DEMAND">
      <value />
    </option>
    <option name="IMPORT_LAYOUT_TABLE">
      <value>
        <package name="android" withSubpackages="true" static="true" />
        <emptyLine />
        <package name="androidx" withSubpackages="true" static="true" />
        <emptyLine />
        <package name="com.android" withSubpackages="true" static="true" />
        <emptyLine />
        <package name="dalvik" withSubpackages="true" static="true" />
        <emptyLine />
        <package name="libcore" withSubpackages="true" static="true" />
        <emptyLine />
        <package name="com" withSubpackages="true" static="true" />
        <emptyLine />
        <package name="gov" withSubpackages="true" static="true" />
        <emptyLine />
        <package name="junit" withSubpackages="true" static="true" />
        <emptyLine />
        <package name="net" withSubpackages="true" static="true" />
        <emptyLine />
        <package name="org" withSubpackages="true" static="true" />
        <emptyLine />
        <package name="java" withSubpackages="true" static="true" />
        <emptyLine />
        <package name="javax" withSubpackages="true" static="true" />
        <emptyLine />
        <package name="" withSubpackages="true" static="true" />
        <emptyLine />
        <package name="android" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="androidx" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="com.android" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="dalvik" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="libcore" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="com" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="gov" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="junit" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="net" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="org" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="java" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="javax" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="" withSubpackages="true" static="false" />
      </value>
    </option>
    <option name="JD_P_AT_EMPTY_LINES" value="false" />
    <option name="JD_DO_NOT_WRAP_ONE_LINE_COMMENTS" value="true" />
    <option name="JD_KEEP_EMPTY_PARAMETER" value="false" />
    <option name="JD_KEEP_EMPTY_EXCEPTION" value="false" />
    <option name="JD_KEEP_EMPTY_RETURN" value="false" />
    <option name="JD_PRESERVE_LINE_FEEDS" value="true" />
  </JavaCodeStyleSettings>
  <JetCodeStyleSettings>
    <option name="PACKAGES_TO_USE_STAR_IMPORTS">
      <value />
    </option>
    <option name="NAME_COUNT_TO_USE_STAR_IMPORT" value="2147483647" />
    <option name="NAME_COUNT_TO_USE_STAR_IMPORT_FOR_MEMBERS" value="2147483647" />
    <option name="CONTINUATION_INDENT_IN_PARAMETER_LISTS" value="false" />
  </JetCodeStyleSettings>
  <XML>
    <option name="XML_ATTRIBUTE_WRAP" value="2" />
    <option name="XML_KEEP_LINE_BREAKS" value="false" />
    <option name="XML_ALIGN_ATTRIBUTES" value="false" />
    <option name="XML_SPACE_INSIDE_EMPTY_TAG" value="true" />
    <option name="XML_LEGACY_SETTINGS_IMPORTED" value="true" />
  </XML>
  <ADDITIONAL_INDENT_OPTIONS fileType="java">
    <option name="TAB_SIZE" value="8" />
  </ADDITIONAL_INDENT_OPTIONS>
  <ADDITIONAL_INDENT_OPTIONS fileType="js">
    <option name="CONTINUATION_INDENT_SIZE" value="4" />
  </ADDITIONAL_INDENT_OPTIONS>
  <codeStyleSettings language="JAVA">
    <option name="ALIGN_MULTILINE_FOR" value="false" />
    <option name="CALL_PARAMETERS_WRAP" value="1" />
    <option name="CALL_PARAMETERS_LPAREN_ON_NEXT_LINE" value="true" />
    <option name="METHOD_PARAMETERS_WRAP" value="1" />
    <option name="RESOURCE_LIST_WRAP" value="1" />
    <option name="EXTENDS_LIST_WRAP" value="1" />
    <option name="THROWS_LIST_WRAP" value="1" />
    <option name="EXTENDS_KEYWORD_WRAP" value="1" />
    <option name="THROWS_KEYWORD_WRAP" value="1" />
    <option name="METHOD_CALL_CHAIN_WRAP" value="1" />
    <option name="BINARY_OPERATION_WRAP" value="1" />
    <option name="BINARY_OPERATION_SIGN_ON_NEXT_LINE" value="true" />
    <option name="TERNARY_OPERATION_WRAP" value="1" />
    <option name="TERNARY_OPERATION_SIGNS_ON_NEXT_LINE" value="true" />
    <option name="FOR_STATEMENT_WRAP" value="1" />
    <option name="ARRAY_INITIALIZER_WRAP" value="1" />
    <option name="ASSIGNMENT_WRAP" value="1" />
    <option name="IF_BRACE_FORCE" value="3" />
    <option name="DOWHILE_BRACE_FORCE" value="3" />
    <option name="WHILE_BRACE_FORCE" value="3" />
    <option name="FOR_BRACE_FORCE" value="3" />
    <option name="ENUM_CONSTANTS_WRAP" value="1" />
  </codeStyleSettings>
  <codeStyleSettings language="XML">
    <option name="FORCE_REARRANGE_MODE" value="1" />
    <indentOptions>
      <option name="CONTINUATION_INDENT_SIZE" value="4" />
    </indentOptions>
    <arrangement>
      <rules>
        <section>
          <rule>
            <match>
              <AND>
                <NAME>xmlns:android</NAME>
                <XML_NAMESPACE>^$</XML_NAMESPACE>
              </AND>
            </match>
          </rule>
        </section>
        <section>
          <rule>
            <match>
              <AND>
                <NAME>xmlns:.*</NAME>
                <XML_NAMESPACE>^$</XML_NAMESPACE>
              </AND>
            </match>
            <order>BY_NAME</order>
          </rule>
        </section>
        <section>
          <rule>
            <match>
              <AND>
                <NAME>.*:id</NAME>
                <XML_NAMESPACE>http://schemas.android.com/apk/res/android</XML_NAMESPACE>
              </AND>
            </match>
          </rule>
        </section>
        <section>
          <rule>
            <match>
              <AND>
                <NAME>.*:name</NAME>
                <XML_NAMESPACE>http://schemas.android.com/apk/res/android</XML_NAMESPACE>
              </AND>
            </match>
          </rule>
        </section>
        <section>
          <rule>
            <match>
              <AND>
                <NAME>name</NAME>
                <XML_NAMESPACE>^$</XML_NAMESPACE>
              </AND>
            </match>
          </rule>
        </section>
        <section>
          <rule>
            <match>
              <AND>
                <NAME>style</NAME>
                <XML_NAMESPACE>^$</XML_NAMESPACE>
              </AND>
            </match>
          </rule>
        </section>
        <section>
          <rule>
            <match>
              <AND>
                <NAME>.*</NAME>
                <XML_NAMESPACE>^$</XML_NAMESPACE>
              </AND>
            </match>
            <order>BY_NAME</order>
          </rule>
        </section>
        <section>
          <rule>
            <match>
              <AND>
                <NAME>.*</NAME>
                <XML_NAMESPACE>http://schemas.android.com/apk/res/android</XML_NAMESPACE>
              </AND>
            </match>
            <order>ANDROID_ATTRIBUTE_ORDER</order>
          </rule>
        </section>
        <section>
          <rule>
            <match>
              <AND>
                <NAME>.*</NAME>
                <XML_NAMESPACE>.*</XML_NAMESPACE>
              </AND>
            </match>
            <order>BY_NAME</order>
          </rule>
        </section>
      </rules>
    </arrangement>
  </codeStyleSettings>
  <codeStyleSettings language="kotlin">
    <option name="ALIGN_MULTILINE_PARAMETERS" value="false" />
    <option name="CALL_PARAMETERS_WRAP" value="5" />
    <option name="METHOD_PARAMETERS_WRAP" value="5" />
    <option name="METHOD_PARAMETERS_LPAREN_ON_NEXT_LINE" value="true" />
    <option name="METHOD_PARAMETERS_RPAREN_ON_NEXT_LINE" value="true" />
    <option name="EXTENDS_LIST_WRAP" value="5" />
    <option name="ENUM_CONSTANTS_WRAP" value="5" />
  </codeStyleSettings>
</code_scheme>
```

Reply:
(I've been meaning that add a formatter to the project, so I should do that soon.)